apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.html

```diff
@@ -144,6 +144,10 @@
                 <span class="pull-right">
                         <button  class="btn btn-primary mrgeBtn"  [disabled]="(isDiscountSaveDisable$ | async)" (click)="applyDiscounts()">Apply</button>
                     </span>
+                <!-- <span class="pull-right">
```

Comment:
Consider removing the commented code if not required

Reply:
done